frontendVfsUtils/src/main/java/com/virtuslab/gitmachete/frontend/vfs/utils/MacheteFileType.java

```diff
@@ -0,0 +1,42 @@
+package com.virtuslab.gitmachete.frontend.vfs.utils;
+
+import javax.swing.Icon;
+
+import com.intellij.openapi.fileTypes.LanguageFileType;
+import com.intellij.openapi.fileTypes.PlainTextLanguage;
+import org.checkerframework.checker.nullness.qual.Nullable;
+
+public class MacheteFileType extends LanguageFileType {
+  public static final MacheteFileType INSTANCE = new MacheteFileType();
+
+  public static final String FILE_NAME = "machete";
+
+  protected MacheteFileType() {
+    super(PlainTextLanguage.INSTANCE);
+  }
+
+  @Override
+  public boolean isReadOnly() {
+    return false;
+  }
+
+  @Override
+  public String getName() {
+    return "Machete";
+  }
+
+  @Override
+  public String getDescription() {
+    return "Configuration file for Git Machete";
```

Comment:
Or maybe even `Branch layout file for Git Machete`? So as to not create impression that any other config is stored there

Reply:
right, fyi this can be seen here:

![image](https://user-images.githubusercontent.com/19799111/83420200-92649800-a426-11ea-8d4d-de47d71789d8.png)